### PR TITLE
Feature/polishing styles

### PR DESCRIPTION
**Pull request by zachfedor**
created meetup page and fixed up a bunch of unfinished styles. still a few style issues, see issue tracker for details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Mi casa es su casa. Everyone is welcome to submit issues and/or pull requests. O
 6. _Optional:_ You can also run `compass watch` to work on sass files and auto-compile them
 7. Also in `/web`, run `php -S localhost:8000` to serve up the project
 8. View the project in a browser at that url
+9. _Note:_ In order to populate the events, visit localhost:8000/fetchevents
```
Comment (amayer5125) [inline README.md:24]: will this eventually be a cron job or something?

Comment (brewcore) [inline README.md:24]: Yep, that's the plan.

File: views/index.twig [hunk 2]
```diff
@@ -16,25 +16,17 @@
...
-                        Bacon ipsum dolor amet sirloin ut sausage, ut boudin flank picanha ullamco minim. Pork loin alcatra dolore sirloin. Shoulder rump turducken bresaola drumstick pariatur eiusmod, sunt dolore kielbasa t-bone leberkas. Short loin magna venison doner ball tip. Lorem salami andouille dolore sunt laborum pariatur corned beef biltong pork ut. Biltong cupim tri-tip, boudin irure short loin ad jerky veniam alcatra porchetta in.
-                    </div>
-
-                    <div class="col-sm-4">
-                        <div class="col-sm-12">
-                            <a href="/resources" class="tl-button" >Resources For Meetups</a>
-                        </div>
+                <div class="row flex-row flex-row-wrap">
+                    <div class="col-sm-9">
+                        <p>
```
Comment (brewcore) [inline views/index.twig:21]: Why the whitespace?

File: views/meetup.twig [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+                <p class="tl-section-header">Tuesday, August 18th, 6:30pm</p>
+            </section>
+
+            <div class="container">
+                <div class="row flex-row flex-row-wrap">
+                    <article class="col-md-4 col-xs-12 tl-event-tile">
+                        <h3>What Is It?</h3>
+
+                        <p>A meetup for and by the local tech community. We'll focus strictly on tech - no sales & marketing here. Open to anyone with an open mind.</p>
```
Comment (brewcore) [inline views/meetup.twig:58]: & should be &amp;

**Comment by brewcore** [comment]
I'm going to go ahead and merge this since there aren't any major issues.

**Pull request merged**